docs/developers/contributing.md

```diff
@@ -210,6 +217,11 @@ You can then make a
 
 ## Building the docs
 
+Install the documents tools:
+```sh
+pip install -e ".[doc]"  # (quotes only needed for zsh shell)
```

Comment:
```suggestion
pip install -e ".[docs]"  # (quotes only needed for zsh shell)
```
I just checked and our doc dependencies are actually in the `docs` extra not `doc`